app/models/okc_blotter.rb

```diff
@@ -0,0 +1,5 @@
+module OkcBlotter
+  def self.table_name_prefix
+    'okc_blotter_'
+  end
```

Comment:
specifies the table prefix for all of the models inside this module
(so the table for OkcBlotter::Booking becomes okc_blotter_bookings)